pipes/WDL/tasks/tasks_demux.wdl

```diff
@@ -79,6 +79,48 @@ task samplesheet_rename_ids {
   }
 }
 
+task revcomp_i5 {
+  input {
+    File    old_sheet
+    Boolean revcomp=true
+    String  docker = "quay.io/broadinstitute/py3-bio:0.1.2"
+  }
+  String new_base = basename(basename(old_sheet, '.txt'), '.tsv')
+  command <<<
+    python3 << CODE
+    import csv
+    import Bio.Seq
+    old_sheet = "~{old_sheet}"
+    revcomp = ~{true="True" false="False" revcomp}
+
+    with open(old_sheet, "rt") as inf:
+      with open(new_base+'.tsv', 'wt') as outf:
+        if revcomp:
+          sheet = csv.DictReader(inf, delimiter='\t')
+          writer = csv.DictWriter(outf, sheet.fieldnames, delimiter='\t', dialect=csv.unix_dialect, quoting=csv.QUOTE_MINIMAL)
+          writer.writeheader()
+          for row in sheet:
+            if row.get('barcode_2')
+              and all(s in 'ATGC' for s in row['barcode_2']):
+              row['barcode_2'] = str(Bio.Seq.Seq(row['barcode_2']).reverse_complement())
```

Comment:
Not that biopython increases the size of the docker image much, but builtins could be used for reverse complementing via `bytearray.maketrans()` (ex. [here](https://github.com/broadinstitute/viral-core/blob/612d269e75905e7b747f67ef56ba120d76f2306b/util/illumina_indices.py#L1524-L1529); could extend to ambiguity codes.). Probably faster too, though that's not a concern for sample sheets.

Reply:
well the general purpose py3-bio docker image already had it, so I figured I'd keep it simple